### ISSUE TITLE
Checkbox in Plugin Settings doesn't work

### ISSUE THREAD

**Issue by fippu82** [dbeck121/CPI-Helper-Chrome-Extension#56]
A checkbox can be added to the plugins settings using "type": "checkbox", but it's not fully implemented yet: 

1. Text is not displayed next to checkbox. 
2. Checkbox status is not saved to storage.
3. Checkbox status is not loaded after reopening the settings modal or reloading page.

**Comment by SAPNickYang** [comment]
> A checkbox can be added to the plugins settings using "type": "checkbox", but it's not fully implemented yet:
> 
> 1. Text is not displayed next to checkbox.
> 2. Checkbox status is not saved to storage.
> 3. Checkbox status is not loaded after reopening the settings modal or reloading page.

Hi Fippu82,

I believe I have fixed these issues in the following pull request: 
[#49 Fixing below bug in the plugins.js. #51 Enhancement in Simple Notepad #50](https://github.com/dbeck121/CPI-Helper-Chrome-Extension/pull/50/commits/1a742acf51ca5a4b99e3fbb2c14bf9edba5ed7c9)
The changes have been implemented in the dev branch, but they have not yet been promoted to the main branch.
Maybe that's why you see it's not fixed in your repo.

Regards,
Nick
